lib/models/organisation.js

```diff
@@ -217,98 +217,108 @@ schema.post('init', function handleInit() {
 schema.pre('save', async function preSave(next) {
   const User = getConnection().model('User');
   const Role = getConnection().model('Role');
+  const organisation = this;
+  const objectId = mongoose.Types.ObjectId;
```

Comment:
Better to save letter case and also you can use desctructuring assignment like so: 

```
const { ObjectId } = mongoose.Types;
```

Reply:
I deleted it because it is not necessary